### PR TITLE
fix: inputNumber has 1px difference with input on the baseline

### PR DESCRIPTION
**Pull request by bbb169**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/43528

### 💡 Background and solution

add 'verticalAlign: 'bottom' css style for class ’ant-input-number‘

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x ] Doc is updated/provided or not needed
- [ x] Demo is updated/provided or not needed
- [ x] TypeScript definition is updated/provided or not needed
- [ x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 978286a</samp>

Fix input number alignment with suffix icon. Add `verticalAlign` style to `components/input-number/style/index.ts` to adjust the position of the input number component.

### <samp>🤖 Generated by Copilot at 978286a</samp>

*  Align input number with suffix icon by adding verticalAlign style ([link](https://github.com/ant-design/ant-design/pull/43534/files?diff=unified&w=0#diff-63d0bbc4ed8cb1745da77de552d75aa0fd581ef3eb2d25d0108b2d55eb5f1857R65))

### CODE REVIEW

File: components/input-number/style/index.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ const genInputNumberStyles: GenerateStyle<InputNumberToken> = (token: InputNumbe
         padding: 0,
         border: `${lineWidth}px ${lineType} ${colorBorder}`,
         borderRadius,
+        verticalAlign: 'bottom',
```
Comment (afc163) [inline components/input-number/style/index.ts:65]: 不应该动这个，这个会导致和其他组件对不齐。

Comment (afc163) [reply components/input-number/style/index.ts:65]: 解法在 https://github.com/ant-design/ant-design/issues/43528#issuecomment-1633588749

Comment (afc163) [reply components/input-number/style/index.ts:65]: 图片

Comment (bbb169) [reply components/input-number/style/index.ts:65]: 新解法已提 https://github.com/ant-design/ant-design/pull/43548

**Pull request closed without merging**